flink-runtime/src/test/java/org/apache/flink/runtime/dispatcher/runner/DefaultDispatcherRunnerITCase.java

```diff
@@ -121,18 +120,20 @@ public void setup() {
         partialDispatcherServices =
                 TestingPartialDispatcherServices.builder()
                         .withFatalErrorHandler(fatalErrorHandler)
-                        .build(blobServerResource.getBlobServer(), new Configuration());
+                        .build(
+                                blobServerExtensionWrapper.getCustomExtension().getBlobServer(),
+                                new Configuration());
     }
 
-    @After
-    public void teardown() throws Exception {
+    @AfterEach
+    void teardown() throws Exception {
         if (fatalErrorHandler != null) {
             fatalErrorHandler.rethrowError();
         }
     }
 
     @Test
-    public void leaderChange_afterJobSubmission_recoversSubmittedJob() throws Exception {
+    void leaderChange_afterJobSubmission_recoversSubmittedJob() throws Exception {
         try (final DispatcherRunner dispatcherRunner = createDispatcherRunner()) {
```

Comment:
just wonder whether we need `dispatcherRunner` here? It seems it is not used anywhere

Reply:
We does need this `dispatcherRunner` as firstDispatcherGateway depend on and connect to it.